lmp/bb-config.sh

```diff
@@ -218,14 +218,14 @@ EOFEOF
 fi
 
 cat << EOFEOF >> conf/auto.conf
-# get build stats to make sure that we use sstate properly
-INHERIT += "buildstats buildstats-summary"
-
 # archive sources for target recipes (for license compliance)
 INHERIT += "archiver"
 COPYLEFT_RECIPE_TYPES = "target"
 ARCHIVER_MODE[src] = "original"
 ARCHIVER_MODE[diff] = "1"
+
+# create SPDX (SBOM) documents
+INHERIT += "create-spdx"
```

Comment:
The problem here, with the removal and addition, is that it will affect every factory we have, and they are all based on different LmP revisions (some which might not have this create-spdx class). Do you know if bitbake will complain when this class is not available?

Reply:
I did a quick test here on kiskstone with  ` INHERIT += "create-spdx1"` and bitbake didn't complain. Maybe on old version we get the same but need to check, at last on dunfell.

Reply:
Unfortunately doesn't work on older versions
```
builder@c9847f2d25db:/lmp/build-intel-corei7-64-lmp$ bitbake lmp-base-console-image -p
ERROR: Unable to start bitbake server (None)
ERROR: Server log for this session (/lmp/build-intel-corei7-64-lmp/bitbake-cookerdaemon.log):
--- Starting bitbake server pid 274 at 2022-11-17 16:44:55.702530 ---
ERROR: ParseError in configuration INHERITs: Could not inherit file classes/create-spdx.bbclass
```

So it need to be conditioned with something like
```
if [ -f layers/openembedded-core/meta/classes/sstate.bbclass ] ; then
    echo 'INHERIT += "create-spdx"' >> conf/auto.conf
fi
```

Reply:
> I did a quick test here on kiskstone with ` INHERIT += "create-spdx1"` and bitbake didn't complain. Maybe on old version we get the same but need to check, at last on dunfell.

Don't know why but this is not true! checking again on kiskstone and it fails as in dunfell
```
ERROR: ParseError in configuration INHERITs: Could not inherit file classes/create-spdx1.bbclass
```

Reply:
There are some movements upstream to backport the `create-spdx` to the LTS dunfell branhc that will be a reality very soon.

Reply:
The problem for us is that we need support in OE and then make sure that every factory is using an updated lmp release.

Reply:
I understand and that is the reason to enable the spdx only when the create-spdx.bbclass exist on the layer.

Reply:
Yup, sounds good.